bubbletea/run.sh

```
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Octofree Monitor Launcher
+# Quick script to build and run the Bubbletea TUI monitor
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)